cmd/agentbasedinstaller/register.go

```diff
@@ -192,7 +192,7 @@ func getReleaseVersionAndCpuArch(log *log.Logger, releaseImage string, releaseMi
 		return "", "", archError
 	}
 
-	return version, cpuArch, nil
+	return version, cpuArch[0], nil
```

Comment:
Should ensure cpuArch is not empty just to be safe?

Reply:
Yeah, just look above at the discussion with Paul. It's not really our piece of code and it has no tests, so I agree there should be _some_ handler but I could as well write
```
if cpuArch == nil {
  panic()
}
```